composer.json

```diff
@@ -12,9 +12,10 @@
         }
     ],
     "require": {
-        "php": ">=7.1.0",
+        "php": "^7.1",
```

Comment:
Nope. I think the "php": ">=7.1.0" is more correct.
When PHP 8 is released, you want your package to be compatible (not to throw an error because it is "php 7 only")